build/ToolsetPackages/project.lock.json

```diff
@@ -30,7 +30,7 @@
         }
       },
       "Microsoft.DiaSymReader.Native/1.2.0-rc": {},
-      "Microsoft.Net.Compilers/1.1.0-beta1": {},
+      "Microsoft.Net.Compilers/1.1.0-beta1-20150727-01": {},
```

Comment:
I was told explicitly, if my building Roslyn causes changes to project.lock.json, I was supposed to check them in.